doc/01-About.md

```diff
@@ -0,0 +1,20 @@
+# Icinga DB
+
+The Icinga DB web module is the way to go, to interact with the new database and to use the benefits it brings.
+
+This module reimplements and aims to replace the old monitoring module.
+It keeps all functionality from its predecessor and adds new quality of life features, such as:
+- bulk operations: set entire host groups into downtime with one operation
+- modals instead of tabs to speed up the workflow
+- a view switcher, that enables the user to choose how much detail should be displayed
+- the status bar now checks for set filters
+
+On top it offers a modern and streamlined design to give an even clearer and more concise overview over the most important points in your environment.
+
```

Comment:
Once https://github.com/Icinga/icingaweb2-module-icingadb/pull/1/files#r366814795 is fixed, please change its content here as well.

Reply:
aye!